src/Symfony/Component/AssetMapper/Compiler/CssAssetUrlCompiler.php

```diff
@@ -38,23 +37,29 @@ public function compile(string $content, MappedAsset $asset, AssetMapperInterfac
     {
         return preg_replace_callback(self::ASSET_URL_PATTERN, function ($matches) use ($asset, $assetMapper) {
             try {
```

Comment:
It looks like we can ignore this. In practice, our `preg_replace_callback()` will return a string. But if there is some change we want to make to satisfy psalm, we can.